models/Post.php

```diff
@@ -112,12 +113,16 @@ class Post extends Model
      */
     public function filterFields($fields, $context = null)
     {
+        $user = BackendAuth::getUser();
+        
+        if ($context == 'create') {
+            $fields->user->value = $user->id;
```

Comment:
I also tried `$fields->user->defaults = $user->id;` but that didn't seem to work?

Reply:
@rubenvanerk This line is fine, however, can you make sure that this line does not override any selection made in the form?

Reply:
@rubenvanerk it should be `$fields->user->default` not `defaults`. I don't think it's a good idea to change the value here, if you were going to change the value itself it should be done in beforeSave only if no other value has been selected.

Reply:
I think I tried `default` too but didn't work. When I dumped the field I saw `defaults` which was filled with the `default` attribute in the yaml file. That's why I tried `defaults`. 
Anyway, the beforeSave method, it won't be transparent to the user what value will be used, I think.

Reply:
I just checked and `$fields->user->default` does not work to set the default value in the form.
If there's a better way to do this I'd love to know. But I think the default value should be reflected in the form so the form doesn't deviate from what is actually saved to the database.

Reply:
@LukeTowers I agree with @rubenvanerk that it should be transparent that it's defaulting to the current user, however, @LukeTowers suggestion could be met by setting the `empty` value of the publisher dropdown to be "Current user", and then `beforeSave` could replace the empty value with the authenticated user ID.

Reply:
@bennothommo I just pushed a commit that fixes the issue and complies with @LukeTowers suggestion.